frontend/nginx/nginx.conf

```diff
@@ -23,9 +22,10 @@ http {
     keepalive_timeout  65;
     gzip  on;
     include /etc/nginx/conf.d/*.conf;
+    include /etc/nginx/sites-available/*;
```

Comment:
What is this line added for?

Reply:
That looks like a left over, I will check and likely be able to remove it.